exe/3scale-copy

```diff
@@ -40,40 +40,59 @@ def compare_hashes(first, second, keys)
   keys.map{ |key| first.fetch(key) } == keys.map{ |key| second.fetch(key) }
 end
 
+def fetch_option(options, key)
+  options.fetch(key) { raise OptionParser::MissingArgument, key }
+end
+
+def provider_key_from_url(url)
+  url[/\w*@/][0..-2]
```

Comment:
on second thought... we probably should allow ThreeScale::API to accept whole uri containing the provider key